browser/ui/views/sidebar/sidebar_container_view.cc

```diff
@@ -179,8 +158,8 @@ void SidebarContainerView::UpdateBackgroundAndBorder() {
 void SidebarContainerView::AddChildViews() {
   sidebar_control_view_ =
       AddChildView(std::make_unique<SidebarControlView>(browser_));
-  sidebar_panel_webview_ =
-      AddChildView(std::make_unique<SidebarPanelWebView>(browser_->profile()));
+
+  ReorderChildView(side_panel_, 1);
```

Comment:
If we add `sidebar_control_view_` at index `0`, then maybe we don't need to reorder `side_pane_`?
```
  sidebar_control_view_ =
      AddChildViewAt(std::make_unique<SidebarControlView>(browser_), 0);
```